Pod/Classes/SEGAdobeIntegration.h

```diff
@@ -9,14 +9,27 @@
 #import <Foundation/Foundation.h>
 #import <Analytics/SEGIntegration.h>
 #import <AdobeMobileSDK/ADBMobile.h>
+#import "ADBMediaHeartbeat.h"
+#import "ADBMediaHeartbeatConfig.h"
 
+@protocol SEGADBMediaHeartbeatFactory <NSObject>
+- (ADBMediaHeartbeat *_Nullable)createWithDelegate:(id _Nullable)delegate andConfig:(ADBMediaHeartbeatConfig *_Nonnull)config;
+;
+@end
+
+
+@interface SEGRealADBMediaHeartbeatFactory : NSObject <SEGADBMediaHeartbeatFactory>
```

Comment:
This should definitely be private. It's only used in the SEGAdobeIntegration.m file